app/scripts/controllers/transactions/index.js

```diff
@@ -1090,4 +1072,43 @@ export default class TransactionController extends EventEmitter {
       }
     }
   }
+
+  _trackTxEvent(txMeta, event, extraOpts = {}) {
```

Comment:
let's add some documentation to this method, and my request would be not to use the tx shorthand and not to hide the metrics portion of the 'event' considering events mean a few things in the controllers portion of our codebase ( something like trackTransactionMetricsEvent is just verbose enough to describe what we're doing to a newcomer. )

Reply:
function name updated and documentation added